tests/unit_tests/pyanaconda_tests/modules/timezone/test_module_timezone.py

```diff
@@ -290,19 +295,24 @@ def test_install_with_tasks_configured(self, publisher):
         )
 
         task_classes = [
+            ConfigureHardwareClockTask,
             ConfigureTimezoneTask,
             ConfigureNTPTask,
         ]
         task_paths = self.timezone_interface.InstallWithTasks()
         task_objs = check_task_creation_list(task_paths, publisher, task_classes)
 
+        # ConfigureHardwareClockTask
+        obj = task_objs[1]
+        assert obj.implementation._is_utc is True
+
         # ConfigureTimezoneTask
-        obj = task_objs[0]
+        obj = task_objs[1]
```

Comment:
You have twice `[1]` but no `[0]` here :)

Reply:
Oops, thanks!